frontend/src/components/assessments/student-experience/NavButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from "react";
+import { Button, HStack, Spacer } from "@chakra-ui/react";
+
+import StudentContext from "../../../contexts/StudentContext";
+
+const NavButtons = (): React.ReactElement => {
+  const { test, currentQuestionIndex, setCurrentQuestionIndex } = useContext(
+    StudentContext,
+  );
+
+  const questionCount = test?.questions.length ?? 0;
+
+  const onFirstQuestion = currentQuestionIndex === 0;
```

Comment:
`onFirstQuestion` makes me think it's a handler LOL what about something like `isFirstQuestion`?

Reply:
oooo! good point :D